src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerMainTest.java

```diff
@@ -134,4 +136,15 @@ void givenCaseCannotBeListedRightAwayAndDirectionsAreBeingMade_thenNoMessagesSen
         assertThat(response.getErrors()).isEmpty();
     }
 
+    @DisplayName("When we have written an adjournment notice and excluded some panel members, add them "
+        + "to the excluded panel members list")
+    @ParameterizedTest
+    @ValueSource(booleans =  {true, false})
+    void givenPanelMembersExcluded_thenAddPanelMembersToExclusionList(boolean areExistingExclusions) {
+        panelMembersGetExcluded(areExistingExclusions);
+
+        assertThat(sscsCaseData.getSchedulingAndListingFields()
+            .getPanelMemberExclusions().getExcludedPanelMembers()).hasSize(areExistingExclusions ? 3 : 2);

```

Comment:
Not a fan of putting logic in the assertion like this. Does this test require a test to prove it is testing properly?
Would much prefer either
1) Two separate @Tests, one each for existing and no existing exclusions
2) Change to @CsvSource and provide boolean areExistingExclusions and int expectedSize

For the same reason, I think you should inline the method panelMembersGetExcluded. The setup for this test is too fragmented and it is hard to tell what is going on without looking in multiple places, and using one's brain to work out what is meant to be happening. 
I know it looks neat like this, but a test should be self-contained, easy to understand, and test one thing.

Reply:
separated it into 2 tests